examples/SOAP_example.ipynb

```diff
@@ -4,78 +4,282 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Using Rascal to Calculate SOAP Vectors\n",
+    "To install rascal:\n",
+    "(NOTE: See the top-level README for the most up-to-date installation instructions.)\n",
+    "+ mkdir ../build \n",
+    "+ cd build\n",
+    "+ cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=ON ..\n",
+    "+ make -j 4\n",
+    "+ make install"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "init_cell": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib notebook\n",
+    "from matplotlib import pylab as plt\n",
+    "\n",
+    "import os, sys\n",
+    "from ase.io import read\n",
+    "sys.path.insert(0,\"../build/\")\n",
```

Comment:
This whole notebook is out of date and will be removed in #172 . Please revert all notebook changes; any example code should go in a small Python file (as per the README and the contribution guidelines).

Reply:
sorry this notebook was not intended to be commited, I ll reset it.